lib/rbac/filterer.rb

```diff
@@ -207,7 +206,7 @@ def search(options = {})
 
       user_filters['match_via_descendants'] = to_class(options[:match_via_descendants])
 
-      exp_sql, exp_includes, exp_attrs = search_filter.to_sql(tz) if search_filter && !klass.try(:instances_are_derived?)
+      exp_sql, exp_includes, exp_attrs = search_filter.to_sql if search_filter && !klass.try(:instances_are_derived?)
```

Comment:
tangent: It would be nice to get away from `instances_are_derived?`. I think there are many cases where we could remove it.